### PR TITLE
Box breadcrumb checkbox

### PR DESCRIPTION
**Pull request by jkusa**
- added knob categories
- added box story
- removed story number prefix
- added breadcrumb story
- added checkbox story

### CODE REVIEW

File: stories/button.stories.js [hunk 1]
```diff
@@ -28,18 +29,20 @@ export const Playground = () => ({
...
-    isActive: boolean('isActive', false),
-    disabled: boolean('disabled', false),
-    isInverse: boolean('isInverse', false),
-    onClick: action('onClick'),
-    size: select('size', SIZES),
-    style: select('style', STYLES),
-    type: select('type', TYPES),
+    isActive: boolean('isActive', false, argument),
+    isInverse: boolean('isInverse', false, argument),
+    onClick: action('onClick', argument),
```
Comment (kevinhinterlong) [inline stories/button.stories.js:40]:
```suggestion
    onClick: action('onClick', attribute),
```
?

Comment (kevinhinterlong) [reply stories/button.stories.js:40]: Not sure how this is supposed to work but it doesn't seem to be doing anything

Comment (jkusa) [reply stories/button.stories.js:40]: yep, good catch

**Review by kevinhinterlong — APPROVED** [review]

File: stories/button.stories.js (hunk 1 above)
Comment (jkusa) [reply stories/button.stories.js]: Actually that is an action instead of knob, let me update that

**Review by mattgraham1995 — APPROVED** [review]

**Pull request merged**